src/kakarot/precompiles/precompiles.cairo

```diff
@@ -76,6 +76,7 @@ namespace Precompiles {
             gas_price=0,
             starknet_contract_address=0,
             evm_contract_address=address,
+            origin=0,
```

Comment:
why is the origin 0 here?

Reply:
no good reason but doesn't have any impact because this is a stopped context, so this value cannot be used. I've put the true origin instead though for clarity